ideascube/catalog_utils/package.py

```diff
@@ -0,0 +1,55 @@
+
+
+import yaml
+import os.path as op
+import zipfile
+import hashlib
+import codecs
+import csv
+
+
+class MediaCenterPackage:
```

Comment:
So we have a class somewhere that represents media content in a zip that we can install on the system and import in the media center.

This here is a class that lets us zip media content into a package from a CSV description.

The two seem awfully related. Should they be merged?